src/vsCodeConfig/verifyTargetFramework.ts

```diff
@@ -20,33 +20,37 @@ export async function verifyTargetFramework(projectLanguage: ProjectLanguage, fo
         const projFileName: string | undefined = projectLanguage === ProjectLanguage.CSharp ? await tryGetCsprojFile(projectPath) : await tryGetFsprojFile(projectPath);
         if (projFileName) {
 
-            const targetFramework: string | undefined = await tryGetTargetFramework(path.join(projectPath, projFileName));
-            if (targetFramework) {
-
-                const tasksResult: IVerifyFrameworkResult | undefined = verifyTasksFramework(folder, targetFramework);
-                const settingsResult: IVerifyFrameworkResult | undefined = verifySettingsFramework(folder.uri.fsPath, targetFramework);
-
-                const mismatchTargetFramework: string | undefined = (tasksResult && tasksResult.mismatchTargetFramework) || (settingsResult && settingsResult.mismatchTargetFramework);
-                if (mismatchTargetFramework) {
-                    context.telemetry.properties.verifyConfigPrompt = 'updateTargetFramework';
-
-                    // This won't handle the case where there are multiple different target frameworks, but it's good enough for the message
-                    const message: string = localize('mismatchTargetFramework', 'The targetFramework "{0}" in your project file does not match the targetFramework "{1}" in your VS Code config.', targetFramework, mismatchTargetFramework);
-                    const update: vscode.MessageItem = { title: localize('updateTargetFramework', 'Update VS Code config') };
-
-                    const result: vscode.MessageItem = await ext.ui.showWarningMessage(message, update, DialogResponses.dontWarnAgain);
-                    if (result === DialogResponses.dontWarnAgain) {
-                        context.telemetry.properties.verifyConfigResult = 'dontWarnAgain';
-                        await updateWorkspaceSetting(settingKey, false, folder.uri.fsPath);
-                    } else if (result === update) {
-                        context.telemetry.properties.verifyConfigResult = 'update';
-                        if (tasksResult) {
-                            await tasksResult.update();
-                        }
-
-                        if (settingsResult) {
-                            await settingsResult.update();
-                        }
+            let targetFramework: string;
+            try {
+                targetFramework = await getTargetFramework(path.join(projectPath, projFileName));
```

Comment:
What's the expected error here?

Reply:
A few possibilities: We fail to find the csproj, we can't read the csproj, we can't find the TargetFramework property